src/Layers/xrRenderGL/glHWCaps.cpp

```diff
@@ -0,0 +1,68 @@
+#include "stdafx.h"
+#pragma hdrstop
+
+#include "Layers/xrRender/HWCaps.h"
+#include "glHW.h"
+
+
+namespace
+{
+u32 GetGpuNum() { return 1; }
+}
+
+
```

Comment:
Please, remove this one empty line. Two empty lines is excessive, one is enough.